gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -664,6 +664,38 @@ gst_tensors_info_get_names_string (const GstTensorsInfo * info)
   return name_str;
 }
 
+/**
+ * @brief GstTensorsInfo represented as a string. Caller should free it.
+ * @param info GstTensorsInfo structure
+ * @return The newly allocated string representing the tensorsinfo. Free after use.
+ */
+gchar *
+gst_tensors_info_to_string (const GstTensorsInfo * info)
+{
+  GString *gstr = g_string_new (NULL);
+  unsigned int i;
+  int limit = info->num_tensors;
+
+  g_string_append_printf (gstr, "Num_Tensors = %u, Tensors = [",
+      info->num_tensors);
+  if (limit > NNS_TENSOR_SIZE_LIMIT) {
+    limit = NNS_TENSOR_SIZE_LIMIT;
+    g_string_append_printf (gstr,
+        "(Num_Tensors out of bound. Showing %d only)", limit);
+  }
+  for (i = 0; i < info->num_tensors; i++) {
```

Comment:
need change `for (i = 0; i < info->num_tensors; i++)` >> `for (i = 0; i < limit; i++)`

Reply:
applied.